sharding-jdbc-example/sharding-example/sharding-raw-jdbc-example/src/main/java/org/apache/shardingsphere/example/sharding/raw/jdbc/JavaRangeConfigurationExample.java

```diff
@@ -47,6 +48,6 @@ public static void main(final String[] args) throws SQLException {
     }
     
     private static CommonService getCommonService(final DataSource dataSource) {
-        return new CommonServiceImpl(new RangeOrderRepositoryImpl(dataSource), new OrderItemRepositoryImpl(dataSource));
+        return new CommonServiceImpl(new RangeOrderRepositoryImpl(dataSource), new OrderItemRepositoryImpl(dataSource),new CountryRepositroyImpl(dataSource));
```

Comment:
JavaRangeConfigurationExample can be removed, we have verify the range feature in Sharding-example.

Reply:
for the raw-jdbc-example, I think add a yamlExample is ok like we have done in transaction-example

Reply:
so I just remove JavaRangeConfigurationExample will be ok?

Reply:
should remove all the range example for broadcast.

Reply:
ok